trunk/SUAVE/Optimization/Nexus.py

```diff
@@ -1,4 +1,4 @@
-## @ingroup Optimization
+ ## @ingroup Optimization
```

Comment:
remove space

Reply:
I'm going to correct this and delete this pull request per our discussion this morning